capywise/lib/main.dart

```diff
@@ -1,29 +1,19 @@
 import 'package:flutter/material.dart';
-import 'core/base_page.dart';
+import 'views/dashboard/dashboard_screen.dart';
```

Comment:
```suggestion
import 'views/dashboard/dashboard_screen.dart';
import 'core/base_page.dart';
```
keep import to base_page. new edits to develop were added using the base page, so rebase on develop to get those edits. we want to keep everything commented out EXCEPT for `home: const LandingPage()` just add a comment including your page specifically `// home: const Dashboard(),` and once we get all of our screens done, we will add the navigation so testing your specific feature is easier. adding it as home is OK for testing ONLY (once we get more features added, then we will start doing verifications where others will checkout and test your branch, but for now keep landingpage as the home)